pkg/logs/logs.go

```diff
@@ -15,47 +18,64 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/status"
 )
 
+const (
+	// key used to display a warning message on the agent status
+	invalidProcessingRules = "invalid_global_processing_rules"
+	invalidEndpoints       = "invalid_endpoints"
+)
+
 var (
 	// isRunning indicates whether logs-agent is running or not
-	isRunning bool
+	isRunning *bool
```

Comment:
I think `isRunning` is not thread safe anymore. Should we use an `atomic int32` ?

Reply:
👍